src/common/utils/invoiceUtils.js

```diff
@@ -1,5 +1,15 @@
+import moment from 'moment';
+
+import { DATE_FORMAT } from '../constants';
+
 export const getInvoiceStatusLabel = ({ status = '' }) => (
   `ui-invoice.invoice.status.${(status || '').toLowerCase()}`
 );
 
 export const formatAmount = (amount = '') => `$${amount.toLocaleString('en')}`;
+
+export const formatDate = (dateString = '') => {
+  const date = moment.utc(dateString || '');
```

Comment:
o_O. Maybe it makes sense to remove default value on the line above

Reply:
yep